Headlines/src/Controllers/TrendingNewsTableViewController.swift

```diff
@@ -12,59 +12,59 @@ class TrendingNewsTableViewController: NewsTableViewController {
 
     var headerView: TopicHeaderView?
     var headerViewModel: TopicHeaderViewModel?
-    
+
     let headerViewHeight: CGFloat = 180.0
     var headerViewHeightConstraint: NSLayoutConstraint?
     var headerViewTopConstraint: NSLayoutConstraint?
-    
+
     private func setupTopicHeaderView() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/adb6600e04f8d78e85d3ed821c2844be0fbd4321/Headlines/src/Controllers/TrendingNewsTableViewController.swift#L20"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 55 lines<br />
<code>function_body_length</code> <code>TrendingNewsTableViewController.swift:20</code></td>
    </tr>
  </tbody>
</table>